### ISSUE TITLE
Fedora 40 Appimage 12.2.0-beta.3

### ISSUE THREAD

**Issue by SteveEbey73742** [WowUp/WowUp#1471]
**Describe the bug**
Won't open on Fedora 40. Gray background then nothing. top shows it runs then closes. 

**To Reproduce**
Auto update is enabled, and I can not disable it. 12.2.0-beta.2 loads fine, but the auto update box appears, and the only recourse is to click on the install button.  Please consider adding a cancel button, or give me a command line option that disables the auto update.

**Expected behavior**
Be able to continue using the app, and not have it crash to desktop. 

**Screenshots**
Unable to capture the screenshot

**Desktop (please complete the following information):**
 - OS: Fedora 40 with Cinnamon DE

**Additional context**
Nothing else to add.

**Comment by jliddev** [comment]
Is this resolved with beta 4+ ?

**Comment by SteveEbey73742** [comment]
Yes, issue resolved. Thank you for the update and the work you have done
for the community of wow.
